lib/neurow/jwt_auth_plug.ex

```diff
@@ -0,0 +1,136 @@
+defmodule Neurow.JwtAuthPlug do
+  require Logger
+
+  import Plug.Conn
+
+  def init(options), do: options
+
+  def call(conn, options) do
+    case jwt_token_from_request(conn) do
```

Comment:
Try to use what seems to be idiomatic of Elixir, but I wonder if I should not use a try / rescue instead.